src/js/tech/tech.js

```diff
@@ -98,7 +98,9 @@ class Tech extends Component {
     this.manualProgress = true;
 
     // Trigger progress watching when a source begins loading
-    this.trackProgress();
+    this.one('ready', () => {
```

Comment:
`one` already does a bind to `this` internally, so the fat arrow function here is redundant.